engines/python/setup/djl_python/rolling_batch/lmi_dist_v2_rolling_batch.py

```diff
@@ -51,7 +53,7 @@ def __init__(self, model_id_or_path: str, properties: dict, **kwargs):
                 "speculate_length"] = self.vllm_configs.speculative_length
             engine_kwargs[
                 "draft_model_tp_size"] = self.vllm_configs.draft_model_tp_size
-        args = EngineArgs(
+        engine_args = EngineArgs(
```

Comment:
should we just remove lmi-dist rolling batch entirely?

Reply:
yes, will remove it entirely in the next PR and will rename lmi_dist_v2 to lmi_dist everywhere in the code. Did not make the change in this PR since it will be easy to switch in case something breaks without much of code review.